service/src/main/kotlin/fi/espoo/evaka/invoicing/service/InvoiceGenerator.kt

```diff
@@ -96,7 +101,8 @@ internal fun generateDraftInvoices(
     operationalDays: OperationalDays,
     absences: List<AbsenceStub> = listOf(),
     freeChildren: List<ChildId> = listOf(),
-    codebtors: Map<PersonId, PersonId?> = mapOf()
+    codebtors: Map<PersonId, PersonId?> = mapOf(),
+    feeThresholds: FeeThresholds
```

Comment:
This sucks since we only need a few fields from FeeThresholds but I thought it's lesser evil than writing a new data class & queries just for this